src/usecases/verifyAdminToken.test.js

```diff
@@ -10,6 +10,7 @@ describe("verifyAdminToken", () => {
   const res = {
     writeHead: jest.fn(() => res),
     end: jest.fn(),
+    setHeader: jest.fn(),
```

Comment:
What do you think about adding a test to check that `verifyAdminToken` calls `refreshToken`?